requirements/system/user-client/functional.tex

```diff
@@ -0,0 +1,50 @@
+\section{Functional Requirements}
+\begin{itemize}
+  \item[UCF10] The user client shall implement all relevant command line options for the specified in Command Line Interface.
+  \item[UCF20] In order to carry out a request, the user client shall authenticate the user with the central server as follows:
+    \begin{enumerate}
+      \item[UCF21] Establish a \gls{SSH} connection to the server.
+      \item[UCF22] Log in the user to the server, using their credentials.
```

Comment:
This requirement is redundant.
The user is already authenticated when he establishes the SSH connection.
There is no need to log in.
I think this requirement should be merged with the previous one.